style.css

```diff
@@ -0,0 +1,105 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --color-primary-text-1: #ff6a01;
+  --color-supporting-text-4: #fef5e0;
+  --color-icons-8: #dbd8d7;
+  --color-description-text-11: #fff;
+  --color-background-1: #1c1a19;
+  --color-nav-bar: #3c3a39;
+}
+
+.header {
+  min-width: 375px;
+  background-color: var(--color-nav-bar, #3d3b39);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2px 24px 2px 12px;
```

Comment:
- [ ] Kindly replace this line with `padding: 12px 24px 12px 12px` to make your design look identical to the figma design as right now the top and bottom width of your navbar is not enough.